src/Service.Tests/GraphQLBuilder/Sql/StoredProcedureBuilderTests.cs

```diff
@@ -54,6 +54,7 @@ public class StoredProcedureBuilderTests
         [DataRow(typeof(DateTime), DATETIME_TYPE, "12/31/2030 12:00:00 AM", false, DisplayName = "DateTime")]
         [DataRow(typeof(DateTime), DATETIME_TYPE, "12/31/2030 12000 AM", true, DisplayName = "DateTime")]
         [DataRow(typeof(DateTimeOffset), DATETIME_TYPE, "11/19/2012 10:57:11 AM -08:00", false, DisplayName = "DateTimeOffset")]
+        [DataRow(typeof(TimeOnly), LOCALTIME_TYPE, "10:57:11.0000", false, DisplayName = "LocalTime")]
```

Comment:
need examples of negative value to see behavior as well as a time over 24 hours i.e. 30 hours

Reply:
Those tests are already there in the GraphQLSupportedTestBase.cs